src/pyuvdata/utils/uvcalibrate.py

```diff
@@ -440,26 +440,28 @@ def uvcalibrate(
             if len(uvcal_freqs_to_keep) < uvcal.Nfreqs:
                 downselect_cal_freq = True
 
-    # check if uvdata.telescope.x_orientation isn't set (it's required for uvcal)
-    uvd_x = uvdata.telescope.x_orientation
+    # check if x_orientation-equivalent in uvdata isn't set (it's required for uvcal)
+    uvd_x = uvdata.telescope.get_x_orientation_from_feeds()
```

Comment:
I think maybe this should move to requiring the feed_array & feed_angle rather than the xorientation.

This code is trying to make sure that we apply the right calibration to the data, for that you need to know which feed the calibrations apply to.

Reply:
So the requirements are indeed set for `feed_array` and `feed_angle` similar to how they used to be set for `x_orientation` -- i.e., required for `UVCal`, and not for `UVData`. Are you suggesting that we additionally require it for `UVData`, or is the above sufficient?

Note that I'm checking for this string mostly for the sake of trying to identically replicate the previous behavior. I could instead check `feed_array` and `feed_angle`, although the logic would mostly just be replicating code that's already in the utilities.

Reply:
I guess I was just worried about the case that `feed_array` and `feed_angle` were set but `get_x_orientation_from_feeds` returned None because it wasn't a n/e arrangement. In that case I don't think we want to ignore what's on uvdata.

Reply:
Okay, I've at least put a check inside of `uvcalibrate` for now so that the x-orientation doesn't get "overwritten" if it returns `None`. There's probably a longer conversation to be had about `uvcalibrate` _should_ behave when the feed information doesn't "match" -- presumably we can chat about this on a telecon in the coming weeks.